### PR TITLE
Add kingom in the nether map by Waterbug (Revived)

### PR DESCRIPTION
**Pull request by Cahrs**
See https://github.com/MT-CTF/maps/pull/53 for more information

Updated with most recent changes.

### CODE REVIEW

File: 18_kingdom_in_the_nether.conf [hunk 1]
```diff
@@ -0,0 +1,12 @@
+team.1.color = red
+team.1 = red
+team.1.pos = 77,-0,77
+team.2.pos = -76,0,-78
+team.2.color = blue
+author = Waterbug
+rotation = x
+h = -139
```
Comment (JostP) [inline 18_kingdom_in_the_nether.conf]: hmmm
```suggestion
h = 139
```

**Comment by JostP** [comment]
Red barrier is placed wrongly... It will have to be rotated for 90°

**Comment by Cahrs** [comment]
Sorry for the late reply, ive gotten distracted by some project ideas and game-jam. Im not sure how to set up maps as i havent looked into that, waterbug would have to attend to that and then i could update the PR

**Comment by Cahrs** [comment]
Thanks for closing.

**Comment by JostP** [comment]
Did you want to close it because Wsterbug banned you?

**Comment by ClobberXD** [comment]
@ JostP Please don't bring offtopic controversial discussions here.

**Pull request closed without merging**